### PR TITLE
add function to desubroutinize

### PR DESCRIPTION
**Pull request by anthrotype**
Uses fontTools subsetter to remove all subroutines. ~~This is done automatically when calling subroutinize on a font that already has subroutines.~~

### CODE REVIEW

**Comment by anthrotype** [comment]
actually it seems that tx doesn't need the input font to be without subroutines, as it's producing identical subroutinized output whether or not the input contained subroutines.
So I have removed the automatic desubroutinization since it's already done by tx itself.
Still, I think it's a good idea to have a `desubroutinize` fuction in here, even only for testing purposes (e.g. check the consistency of the subroutinization itself, by comparing  and checking the desubroutinized versions of the original and subroutinized font are the same)

**Comment by anthrotype** [comment]
I also considered using tx itself with -S option to do the desubroutinization, instead of fonttools subset module, but then I thought that they ought to produce the same output so it shouldn't matter. We can revisit later.

File: src/cffsubr/__init__.py [hunk 1]
```diff
@@ -197,3 +198,49 @@ def subroutinize(
...
+    # the 'desubroutinize' method is dynamically added to the CFF table class
+    # as a side-effect of importing the fontTools.subset.cff module...
+    from fontTools.subset import cff as _
+
+    if not inplace:
+        otf = copy.deepcopy(otf)
+
+    table_tag = _sniff_cff_table_format(otf)
+    try:
+        otf[table_tag].desubroutinize()
```
Comment (khaledhosny) [inline src/cffsubr/__init__.py:242]: `tx` can desubroutinize as well (`tx -cff -S`), no idea if it is any faster.

Comment (anthrotype) [reply src/cffsubr/__init__.py:242]: yes, i know https://github.com/adobe-type-tools/cffsubr/pull/5#issuecomment-635936610

I guess it shouldn't matter in the end, as there's only one correct way to desubroutinize. I guess.

Comment (anthrotype) [reply src/cffsubr/__init__.py:242]: > if it is any faster

possibly, but desubroutinization is mostly meant for testing/debugging.

**Pull request merged**